content/blog/day-2-kubernetes-migrating-eks-nodegroups-with-zero-downtime/index.md

```diff
@@ -0,0 +1,170 @@
+---
+title: "Day 2 Kubernetes - Migrating EKS Node Groups with Zero Downtime"
+authors: ["mike-metral"]
+tags: ["Kubernetes","EKS", "Migrations", "Node Groups"]
+date: "2019-07-18"
+
+meta_image: "eks-migrate-nodegroups.svg"
+---
+
+Managed Kubernetes clusters greatly reduce the overhead required in
```

Comment:
This somehow reads a little incomplete. Do you mean "administering Kubernetes deployments"?

Reply:
I was referring to Managed Kubernetes offerings like EKS. I've changed clusters -> offerings to make this more straight-forward.